setup.py

```diff
@@ -61,5 +61,5 @@
         'structuretoolkit==0.0.11'
     ],
     cmdclass=versioneer.get_cmdclass(),
-
+    package_data={'': ['_data/*.csv']},
```

Comment:
Well, I considered this private to the repo. However, since there is no `__init__` (at least not yet). It will not pop up in the dir of the package.

Reply:
Yes, I also do not expect there to be any `__init__` in the data directory but on the root level of the repository I would prefer to have it as `data` rather than `_data` or `.data`.